types/custom-functions-runtime/index.d.ts

```diff
@@ -48,6 +48,11 @@ declare namespace CustomFunctions {
      * Provides information about the invocation of a custom function.
      */
     interface Invocation {
+        /**
+         * The name of this function.
+         * [Api set: CustomFunctionsRuntime 1.1]
```

Comment:
@alison-mk Is this part of CustomFunctionsRuntime 1.1 or a later version?

Reply:
This is part of CustomFunctionsRuntime 1.1. It was previously undocumented. Noted by customer in invocation.functionName property OfficeDev/office-js-docs-reference#1000.